nft/src/lib.rs

```diff
@@ -1069,4 +1088,15 @@ impl<T: Config> Pallet<T> {
 			false
 		}
 	}
+
+	fn is_registered_enclave(account: &T::AccountId) -> bool {
```

Comment:
Does it make sense to have this function? You can get the same effect by doing the following:
```rust
Self::get_enclave(account).is_some()
```